courses/models_test.py

```diff
@@ -172,6 +202,26 @@ def test_program_learning_techniques():
         assert technique.value.get("image").title == "image-title"
 
 
+def test_program_target_audience():
```

Comment:
this test is pretty similar to `test_program_who_should_enroll` has same docstring as well, what is the real difference between them ?

Reply:
Removed, `test_program_target_audience` is redundant.